package.json

```diff
@@ -107,16 +107,14 @@
     "webdriver-manager": "12.0.6",
     "webpack": "^3.5.4",
     "webpack-bundle-analyzer": "^3.0.2",
+    "@okta/okta-auth-js": "~2.3.1",
```

Comment:
backbone and auth-js/jquery depends on `jquery` hence has to move to devDependency.

Reply:
Probably not a big issue, but we should mention that this library will not work properly when using `npm install --production` or `yarn install --production`.

Reply:
I'm thinking mentioned in the 3.0 release notes so that developers would be aware

Reply:
Should we also add in the README?

Reply:
probably not IMHO

Reply:
🚢

Reply:
This seems odd, as AuthJS only has a devDependency on jQuery. Why would that be causing an install issue for us?